materialize-databricks/driver.go

```diff
@@ -246,6 +252,31 @@ func (d *transactor) Load(it *m.LoadIterator, loaded func(int, json.RawMessage)
 			return fmt.Errorf("converting Load key: %w", err)
 		} else if err := b.loadFile.encodeRow(converted); err != nil {
 			return fmt.Errorf("encoding row for load: %w", err)
+		} else {
+			bindingName := b.target.Identifier
+			if bmx, ok := bindingMinMaxKeys[bindingName]; !ok {
+				bindingMinMaxKeys[bindingName] = minMax{
+					Min: converted[0],
+					Max: converted[0],
+				}
+			} else {
+				switch v := converted[0].(type) {
```

Comment:
See [here](https://github.com/estuary/connectors/blob/96714ea85e5e4ed5405fc6c7cef843d645c4ffde/materialize-sql/templating.go#L103-L119) for what I believe are the full set of types that keys can take. They might be encoded `uint64`, or possibly `float64` (not 100% sure on that) if the otherwise-integer values are really big.